stable/sematext-docker-agent/values.yaml

```diff
@@ -0,0 +1,23 @@
+image:
+  repository: sematext/sematext-agent-docker
+  tag: latest
```

Comment:
For a tag we need a release tag. This is for reproducible. If one does something in testing, then the latest is updated, and then it's deployed to production you end up deploying a different version to production than was tested. This can lead to issues.